saleor/graphql/translations/mutations.py

```diff
@@ -0,0 +1,185 @@
+import graphene
+from django.core.exceptions import ImproperlyConfigured
+
+from ...discount import models as discount_models
+from ...menu import models as menu_models
+from ...page import models as page_models
+from ...product import models as product_models
+from ...shipping import models as shipping_models
+from ..core.mutations import (
+    BaseMutation, ModelMutationOptions, get_model_name, get_output_fields,
+    registry)
+
+
+class BaseTranslateMutation(BaseMutation):
+
+    @classmethod
+    def __init_subclass_with_meta__(
```

Comment:
Why do we need to override this method? Shouldn't this mutation override `ModelMutation` and only have a custom `mutate` function?